docs/content/guides/dagster/how-assets-relate-to-ops-and-graphs.mdx

```diff
@@ -153,3 +153,5 @@ def send_emails_job():
 ```
 
 In this case, the asset - specifically, the table the job reads from - is only used as a data source for the job. It’s not materialized when the graph is run.
+
+You can read more about how this works [here](/concepts/ops-jobs-graphs/graphs#loading-an-asset-as-an-input).
```

Comment:
```suggestion
Refer to the [Graph documentation](/concepts/ops-jobs-graphs/graphs#loading-an-asset-as-an-input) for more info on how this works.
```

Generally, I think we should avoid vague link titles like `here` - it's helpful to provide context so the user has some expectation of where they're going when they click the link.